src/pages/keyblocks/[id].vue

```diff
@@ -5,12 +5,17 @@
 
   <page-header>
     Keyblock
+
+    <template v-if="!keyblockFound">
+      not found
+    </template>
+
     <template #tooltip>
       {{ keyblocksHints.keyblock }}
     </template>
   </page-header>
 
-  <template v-if="!isLoading">
+  <template v-if="!isLoading && keyblockFound">
```

Comment:
In this case wrapping by template just to chain condition it's complicating the readability. It's an extension of an idea I expressed in `src/pages/microblocks/[id].vue` I think it's too much logic in template. I think the better place for logic is in script block where would be nicely explained by naming. I would suggest something like.

````
 <keyblock-details-panel
      v-if="!isDetailsLoading"
      :keyblock-details="keyblockDetails"/>
````

or


````
 <keyblock-details-panel
      v-if="isDetailsLoaded"
      :keyblock-details="keyblockDetails"/>
````

Reply:
This comment also applies here: https://github.com/aeternity/aescan/pull/444#discussion_r1293317241